hyperspace/core/src/packets/utils.rs

```diff
@@ -205,35 +201,41 @@ pub async fn construct_timeout_message(
 	next_sequence_recv: u64,
 	proof_height: Height,
 ) -> Result<Any, anyhow::Error> {
-	let key = if sink_channel_end.ordering == Order::Ordered {
-		let path = get_key_path(KeyPathType::SeqRecv, &packet);
-		apply_prefix(sink.connection_prefix().into_vec(), path)
+	let path_type = if sink_channel_end.ordering == Order::Ordered {
```

Comment:
we don't currently support `Ordered right`?

Reply:
I think we should

Reply:
Ok!